modules/mint-mode-with-removal-of-admin-credential.adoc

```diff
@@ -3,25 +3,16 @@
 // * installing/installing_aws/manually-creating-iam.adoc
 
 [id="mint-mode-with-removal-or-rotation-of-admin-credential_{context}"]
-= Mint Mode with removal or rotation of the admin credential
+= Mint mode with removal or rotation of the admin credential
 
-Currently, this mode is only supported on AWS.
+Currently, this mode is only supported on AWS and GCP.
 
-In this mode, a user installs {product-title} with an `admin` credential just
-like the normal mint mode. However, this mode removes the `admin` credential
-secret from the cluster post-installation.
+In this mode, a user installs {product-title} with an `admin` credential just like the normal mint mode. However, this mode removes the `admin` credential secret from the cluster post-installation.
 
-The administrator can have the Cloud Credential Operator make its own request
-for a read-only credential that allows it to verify if all `CredentialsRequest` objects
-have their required permissions, thus the `admin` credential is not required
-unless something needs to be changed. After the associated credential is
-removed, it can be destroyed on the underlying cloud, if desired.
+The administrator can have the Cloud Credential Operator make its own request for a read-only credential that allows it to verify if all `CredentialsRequest` objects have their required permissions, thus the `admin` credential is not required unless something needs to be changed. After the associated credential is removed, it can be destroyed on the underlying cloud, if desired.
```

Comment:
```suggestion
The administrator can have the Cloud Credential Operator make its own request for a read-only credential that allows it to verify if all `CredentialsRequest` objects have their required permissions, thus the `admin` credential is not required unless something needs to be changed. After the associated credential is removed, it can be deleted or deactivated on the underlying cloud, if desired.
```